modular_chomp/code/game/machinery/food_replicator.dm

```diff
@@ -82,6 +87,9 @@
 			printing = TRUE
 			update_icon()
 
+			if(product_path == /obj/item/weapon/reagent_containers/food/snacks/donkpocket/ascended)
```

Comment:
could even check if any food contains supermatter as reagent, so in case anyone adds some new funny food it's handled the same